blockchain/source/migrations/2_deploy_v1.js

```diff
@@ -0,0 +1,181 @@
+let Multisig = artifacts.require('./MultiSigWallet.sol');
+let OracleMultisig = artifacts.require('./MultiSigWallet.sol');
+let SNM = artifacts.require('./SNM.sol');
+
+let Market = artifacts.require('./Market.sol');
+let Blacklist = artifacts.require('./Blacklist.sol');
+let ProfileRegistry = artifacts.require('./ProfileRegistry');
+let Oracle = artifacts.require('./OracleUSD.sol');
+let GateKeeper = artifacts.require('./SimpleGatekeeperWithLimit.sol');
+let GateKeeperLive = artifacts.require('./SimpleGatekeeperWithLimitLive.sol');
+let AddressHashMap = artifacts.require('./AddressHashMap.sol');
+let TestnetFaucet = artifacts.require('./TestnetFaucet.sol');
+
+const TruffleConfig = require('../truffle');
+
+const ContractRegistry = require('../migration_utils/address_hashmap');
+
+let freezingTime = 60 * 15;
+let actualGasPrice = 3000000000;
+let benchmarksQuantity = 12;
+let netflagsQuantity = 3;
+
+function MSRequired (network) {
+    if (network === 'rinkeby' || network === 'private' || network === 'dev_main' || network === 'dev_side') {
+        return 1;
+    } else {
+        return 5;
+    }
+}
+
+function MSOwners (network, accounts) {
+    if (network === 'dev_main' || network === 'dev_side') {
+        return accounts;
+    } else {
+        accounts = [
+            '0xdaec8F2cDf27aD3DF5438E5244aE206c5FcF7fCd',
```

Comment:
inactive owner address

Reply:
it was there previously, so I didn't change it